.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: "[Feature]"
+labels: enhancement
+assignees: Hmerin
+
+---
+
+**My feature request is related to**
+
+- [x] Topics
+- [ ] Examples
+- [ ] Exercises
+- [ ] Repository structure
+- [ ] Translations
+- [ ] Other
+
+**Is your feature request related to a problem? Please describe.**
+
+- [ ] Yes
+- [x] No
+
+A clear and concise description of what the problem is. Ex. I don't find the repo structure easy to [...]
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
```

Comment:
This is part of the Github template, we can edit this part to something more clear.

If you want something solved you have to explain that before this section, this part is to suggest a solution. Btw I'm realising that is not a place to explain the problem itself so I will add that.

Reply:
@Gracrys I just remove that part because if it is a problem to be solve it should be an issue. Instead of solutions, I focus on the explanation of the feature itself.

Reply:
I think is ok if it is based on github template